src/c.rs

```diff
@@ -275,10 +316,14 @@ impl LedCanvas {
 
 #[link(name = "rgbmatrix")]
 extern "C" {
-    pub fn led_matrix_create_from_options(
-        options: *const LedMatrixOptions,
-        argc: *mut c_int,
-        argv: *mut *mut *mut c_char,
+    // pub fn led_matrix_create_from_options(
+    //     options: *const LedMatrixOptions,
+    //     argc: *mut c_int,
+    //     argv: *mut *mut *mut c_char,
+    // ) -> *mut LedMatrix;
```

Comment:
Intentionally keeping around dead code?

Reply:
Fair point... might as well delete the unused C APIs.